src/Metadata/class-post-builder.php

```diff
@@ -22,7 +22,7 @@
  *
  * @since 3.4.0
  *
- * @phpstan-import-type ParselyOptions from Parsely
+ * @phpstan-import-type Parsely_Options from Parsely
```

Comment:
Using `_` in naming which is similar to how we name other things. This has been done to some other types also.